ChangeLog

```diff
@@ -1,3 +1,90 @@
+	- Use c11 standard See: #1055 (#1056) (Kang Lin)
+	- Reduce usage of TURN_NO_PROMETHEUS (#1023) (Pavel Punsky)
+	- Remove unnecessary declaration from header file (#1052) (Yoshiki Kadoshita)
+	- Fix compilation warnings for functions with no arguments (#1053) (Pavel Punsky)
+	- Support Windows MSVC (#855) (Kang Lin)
+	- Fix resource leaks (#1048) (Gregor Jasny)
+	- Update turnserver.conf (#1009) (Hyorin Choi)
+	- Fix warnings (#1046) (Gregor Jasny)
+	- Backlog fifo (#1029) (Scott Godin)
+	- Change rpm systemd service type from notify to exec (#1043) (Gustavo Garcia)
+	- Add missing comma (#1041) (Gregor Jasny)
+	- Fix off-by-one when terminating gcm_nonce (#1039) (Gregor Jasny)
+	- Use %zu format specifier for size_t (#1040) (Gregor Jasny)
+	- Fix variable argument handling (#1042) (Gregor Jasny)
+	- Cleanup openssl initialization (#1012) (Pavel Punsky)
+	- fuzzing support (#982) (Arjun)
+	- created netengine.c get_relay_server utility method to reduce code duplication (#1032) (Scott Godin)
+	- fix bug in calls to ssl_read and ssl_send where extra verbose flag goes missing (#1033) (Scott Godin)
+	- ignore raw UDP if no_udp is enabled (#1031) (Scott Godin)
+	- Fix 2 warnings in code (#1027) (Pavel Punsky)
+	- Sanitize DB connection string before printing to log (#1020) (Pavel Punsky)
+	- Better detect SCTP protocol (#1016) (Pavel Punsky)
+	- Redis memleaks and socketleaks (#1015) (Thibaut ACKERMANN)
+	- Fix : Issue 51563 in oss-fuzz (#1010) (Arjun)
+	- Fix multiple warnings in libtelnet.c file (#1011) (Pavel Punsky)
+	- Update libtelnet to 0.23 (portability issues) (#1005) (Pavel Punsky)
+	- Remove debug publish to redis (#1008) (Pavel Punsky)
+	- Cleanup unused include of header files (#1004) (Pavel Punsky)
+	- Use single SSL_CTX for DTLS support (#996) (Pavel Punsky)
+	- Malformed response to mobility refresh request (#1006) (Tom Bevan)
+	- Silence warnings by converting STRCPY to strncpy calls (#995) (Pavel Punsky)
+	- Build CI with prometheus support (#999) (Pavel Punsky)
+	- Replace references to non-existent pdf file with links (#1002) (Pavel Punsky)
+	- Fix TLS1.3 support (Pavel Punsky)
+	- Use a single SSL context object (#989) (Pavel Punsky)
+	- Use epoll for promhttp server if supported.  (#997) (Joachim Bauch)
+	- Fix issues reported by cppcheck (#987) (Pavel Punsky)
+	- Replace bcopy with memcpy (#991) (Pavel Punsky)
+	- Add CI tests in older ubuntu version (#981) (Gustavo Garcia)
+	- Replace bzero with memset (#986) (Pavel Punsky)
+	- Fix a warning (#988) (Pavel Punsky)
+	- Improve openssl3 and FIPS support (#955) (Pavel Punsky)
+	- Preserve file timestamps when using install(1) (#983) (Robert Scheck)
+	- Fix renegotiation flag for older version of openssl (#978) (Pavel Punsky)
+	- Improve GitHub release notes of Docker image (tyranron)
+	- Improve Docker CI pipeline (tyranron)
+	- Upgrade Docker image to 4.6.0 Coturn version (tyranron)
+	- Fix ChangeLog for 4.6.0 release (#976) (tyranron)
```

Comment:
@eakraly changes regarding Docker image may be removed too. We track them in a separate changelog. Here is better to keep changes regarding Coturn itself.